tests/test_helpers.py

```diff
@@ -0,0 +1,80 @@
+"""
+This module contains unit tests for the `helpers.py` module.
+"""
+
+import os
+import shutil
+
+from tests.helpers import setup_test_directory
+
+
+def test_setup_test_directory_creates_directory():
```

Comment:
Would be obsolete using pytest

Reply:
Not necessarily, or am I missing something. Isn't it still worth testing?

Reply:
Hm in theory you can still tests this.
You are right.